src/ResourceManager/Websites/Commands.Websites/Cmdlets/DeploymentSlots/SetAzureWebAppSlot.cs

```diff
@@ -80,8 +80,12 @@ public class SetAzureWebAppSlotCmdlet : WebAppSlotBaseCmdlet
         [ValidateNotNullOrEmpty]
         public bool Use32BitWorkerProcess { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = "Destination slot name for auto swap")]        
+        [Parameter(Mandatory = false, HelpMessage = "Destination slot name for auto swap")]
+
```

Comment:
@LukaszStem remove new line (so that the property and attribute are together), add a new line after property to separate it from next attribute